integrations/salesviewer/gitbook-manifest.yaml

```diff
@@ -0,0 +1,43 @@
+name: salesviewer
+title: SalesViewer
+icon: ./assets/icon.png
+description: Track and identify GitBook page visitors with Salesviewer analytics.
+previewImages:
+  - ./assets/salesviewer-preview.png
+externalLinks:
+  - label: Documentation
+    url: https://www.gitbook.com/integrations/salesviewer
```

Comment:
Hi @KevinRohn - can you please update this to a link to the salesviewer documentation?

Reply:
I couldn't find a dedicated doc page, so I believe the correct link is to the Help Center. There, you can quickly find information on how to get the Tracking/Analytics code.